pkg/assetvalidator/validateAssets.go

```diff
@@ -9,22 +9,42 @@ import (
 	"github.com/asecurityteam/nexpose-asset-producer/pkg/domain"
 )
 
+// NexposeScanType is a type alias to signify type of scan according to Nexpose
+type NexposeScanType string
+
+const (
+	automated NexposeScanType = "Automated"
+	manual    NexposeScanType = "Manual"
+	scheduled NexposeScanType = "Scheduled"
+	agent     NexposeScanType = "Agent"
+)
+
+// ScanType is a type alias that generifies a scan to either local or remote
+// The purpose is to not expose Nexpose specifics further down the pipeline
+type ScanType string
+
+const (
+	local   ScanType = "local"
+	remote  ScanType = "remote"
+	unknown ScanType = "unknown"
+)
+
 // NexposeAssetValidator is used to validate a list of retrieved Assets
 type NexposeAssetValidator struct {
 }
 
 // ValidateAssets takes in a list of assets from Nexpose, and returns a list of valid Asset events, and a list of errors based
 // on defined validation rules
-func (v *NexposeAssetValidator) ValidateAssets(ctx context.Context, assets []domain.Asset, scanID string) ([]domain.AssetEvent, []error) {
+func (v *NexposeAssetValidator) ValidateAssets(ctx context.Context, assets []domain.Asset, scanInfo domain.ScanInfo) ([]domain.AssetEvent, []error) {
 	assetEventListList := []domain.AssetEvent{}
 	errorList := []error{}
 	for _, asset := range assets {
-		scanTime, err := v.getScanTime(asset, scanID)
+		scanTime, scanType, err := v.getScanTimeAndScanType(asset, scanInfo)
```

Comment:
task: as far as I can tell, there is no need for `getScanTimeAndScanType` to return the scanType, as this type is determined by the `NexposeScanType`. 

While this function can return "unknown", this is only for error conditions, in which the error value is used and not any information on the asset, including its scan type. I recommend removing scanType from the function's return values, as well as the "unknown" value from the list of valid constants for ScanType.